website/content/api-docs/secret/transit.mdx

```diff
@@ -91,6 +91,155 @@ $ curl \
     http://127.0.0.1:8200/v1/transit/keys/my-key
 ```
 
+## Import Key
+
+This endpoint imports existing key material into a new transit-managed encryption key.
+To import key material into an existing key, see the `import_version/` endpoint.
+
+### Parameters
+
+- `name` `(string: <required>)` – Specifies the name of the encryption key to
+  create. This is specified as part of the URL.
+
+- `ciphertext` `(string: <required>)` - A base64-encoded string that contains
+two values: an ephemeral AES key wrapped using the wrapping key
+returned by Vault and the encryption of the import key material under the
+provided AES key. The wrapped AES key should be the first 512 bytes of the
+ciphertext, and the encrypted key material should be the remaining bytes.
+
+- `hash_function` `(string: "SHA-256")` - The hash function used for the
```

Comment:
Comparing against the code looks like we are expecting non-dashed versions of the hash function names such as 
`SHA256`...